avocado/plugins/testlogs.py

```diff
@@ -63,3 +63,39 @@ def post(self, job):
                 except (FileNotFoundError, PermissionError) as error:
                     LOG_UI.error('Failure to access log file "%s": %s',
                                  path, error)
+
+
+class TestLogging(ResultEvents):
+
+    description = "Test logs for Job"
```

Comment:
Maybe be nice to document here and/or any other place that this is specific for nrunner.